Emby.Server.Implementations/Localization/LocalizationManager.cs

```diff
@@ -260,6 +264,18 @@ public IEnumerable<ParentalRating> GetParentalRatings()
                 }
             }
 
+            // Remove prefix country code to handle "DE-18"
+            index = rating.IndexOf('-', StringComparison.Ordinal);
+            if (index != -1)
+            {
+                var trimmedRating = rating.AsSpan(index).TrimStart('-').Trim();
+
+                if (!trimmedRating.IsEmpty)
+                {
+                    return GetRatingLevel(trimmedRating.ToString());
+                }
+            }
+
             // TODO: Further improve by normalizing out all spaces and dashes
```

Comment:
Post-overhaul is this TODO still relevant?

Reply:
You never know what weird stuff people put into their NFOs...